morphir/runtime/src/org/finos/morphir/runtime/quick/EvaluatorQuick.scala

```diff
@@ -155,7 +156,12 @@ object EvaluatorQuick {
         Data.Int32(value.toInt)
       case (Concept.Int64, RTValue.Primitive.Int(value)) =>
         Data.Int64(value.toLong)
-
+      case (Concept.Order, RTValue.ConstructorResult(FQString("Morphir.SDK:Basics:gT"), List())) =>
+        Data.Order(1)
+      case (Concept.Order, RTValue.ConstructorResult(FQString("Morphir.SDK:Basics:lT"), List())) =>
+        Data.Order(-1)
+      case (Concept.Order, RTValue.ConstructorResult(FQString("Morphir.SDK:Basics:eQ"), List())) =>
```

Comment:
Why `eQ` and not `EQ`?

Reply:
This is using a FQString, which is an extractor intended for easy matching, given that FQName(someString) doesn't work in match statements. FQString uses toString (and then falls back  to matching on that string); given that the default is camelCase for the localName, EQ becomes eQ.

Potentially that extractor could be broken up for different formats, but that seemed like a lot of additional complexity for minor gain.

Reply:
Well that FQString extractor is wrong and not following Morphir name handling conventions. That is going to add un-needed confusion.

Reply:
I only feel comfortable merging this PR if there is a fast follower to correct `FQString`. The Morphir naming conventions are unique enough, adding an arbitrary divergence will only complicate things now and in the future.

Reply:
I think it's correct when used for functions, but not for constructors/types - looking over use cases, tho, it is mostly (only?) being used for constructors/types. That will only take a moment to change.

Reply:
I fixed it, fast enough follow? :)